internal/tiltfile/include.go

```diff
@@ -0,0 +1,67 @@
+package tiltfile
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"go.starlark.net/starlark"
+)
+
+type loadCacheEntry struct {
+	status  loadStatus
+	exports starlark.StringDict
+	err     error
+}
+
+type loadStatus int
+
+const (
+	loadStatusNone loadStatus = iota
+	loadStatusExecuting
+	loadStatusDone
+)
+
+// Tiltfiles support both load() and include().
```

Comment:
As of this PR it only supports `include()`. Is this comment forward looking?

Reply:
did you see `TestLoadFunction`?

Reply:
Hmm yes I see that now, but I was looking for `load` in the list of built-in functions https://github.com/windmilleng/tilt/pull/1996/files#diff-187cac6214111b4c1831b65825c135f8R180